CMakeLists.txt

```diff
@@ -251,6 +251,15 @@ if(UNIX)
       set(OPENSSL_INCLUDE_DIR /usr/local/opt/openssl/include)
     endif()
   endif()
+  # Some Linux distributions provide an OpenSSL-1.0 package alongside 1.1+
+  if (UNIX)
+    if (NOT OPENSSL_ROOT_DIR)
```

Comment:
Checking for version number can also be useful:
`if (NOT OPENSSL_ROOT_DIR OR NOT OPENSSL_VERSION VERSION_LESS "1.1")`